pkg/option/config.go

```diff
@@ -802,6 +802,15 @@ const (
 
 	// K8sEnableAPIDiscovery
 	K8sEnableAPIDiscovery = "enable-k8s-api-discovery"
+
+	// LBMapEntriesName configures max entries for BPF lbmap.
+	LBMapEntriesName = "bpf-lb-map-max"
+
+	// LBMapMinEntries defines the minimum lbmap limit.
+	LBMapMinEntries = 1 << 10 // 1Ki entries
```

Comment:
Where does the min entries limit come from? I think some users might want to go below this limit (think of embedded devices with very limited memory).

Reply:
I discussed this on slack with @aanm. This is similar to the limits on CT/NAT table - [LimitTable{Max,Min}](https://github.com/cilium/cilium/blob/master/pkg/option/config.go#L490-L494). We discussed that this could be a good starting point. But if it makes sense like for the case you mentioned we can decide on a more suitable limit.

Reply:
Can we then just remove those artificial limits? I think the CT/NAT limits might be used by the dynamic scaling.

Reply:
@fristonio why not just check for invalid configs like map size of 0 and the rest is up to the user?

Reply:
@borkmann @brb let's address in a different PR, this was essentially introduced by @joestringer  https://github.com/cilium/cilium/commit/ec07900a49bf96b2f47b6ecbc0b4c22c7ca4de46#diff-b72a2fd7a91a97fb4b234e7919334832R1090

Reply:
Let's just remove the checks in this PR.

Reply:
Cool, I will remove the checks on `lbmap` size from this PR and will remove the defaults for these sizes from `pkg/defaults`.